sdks/java/io/kinesis/build.gradle

```diff
@@ -50,6 +50,7 @@ dependencies {
   testCompile library.java.powermock
   testCompile library.java.powermock_mockito
   testCompile "org.assertj:assertj-core:3.11.1"
+  testCompile 'org.testcontainers:localstack:1.11.2'
```

Comment:
Very good point. I copied it from DynamoDB tests. Thanks!

Reply:
Since `localstack` is going to be used in several modules, it would make sense to expand it as a separate library in `BeamModulePlugin` (like, `library.java.powermock` for example).

Reply:
Yeah, I'll do that.